FEC_Puzzle/FEC_Puzzle/FEC_Puzzle.ino

```diff
@@ -0,0 +1,40 @@
+#define LedPin 13
+#define SolenoidPin 12
+
+int HandDetection = 700;
+int HandsPin = A3;
+int CoilPin = A5;
+
+void setup() {
+  Serial.begin(9600); //  setup serial
+  pinMode(LedPin, OUTPUT); //led
+  pinMode(SolenoidPin, OUTPUT); //solenoid
+}
+
+void loop() {
+  HandPuzzle();
+  CoilPuzzle();
+}
+
+void HandPuzzle() {
+  int HandsValue = analogRead(HandsPin);  // read the input pin
+  //Serial.println(ReadValue); // debug value
+  if (HandsValue < HandDetection) {
+    digitalWrite(LedPin, HIGH);
```

Comment:
Ik zou hier nog een comment neer zetten of dit dan WEL of NIET de hand gedecteert heeft